src/server/views/dashboard/macros.njk

```diff
@@ -38,10 +38,10 @@
     <div class="govuk-error-summary__body">
       <ul class="govuk-list govuk-error-summary__list">
         <li>
-          {% if error?.href %}
+          {% if not error?.href %}
             <a href="{{ error.href }}">{{ error.text }}</a>
           {% else %}
-            {{ error.text if error.text else error }}
+            {{ error.text }}
           {% endif %}
```

Comment:
Revert. I fixed this in #547. This change reverts it back to "if not error.href, render a link with href" which doesn't make any sense